### PR TITLE
 Coinbase transfers between account and subaccounts

### PR DESCRIPTION
**Pull request by EdNoepel**
Implemented API to get profiles and transfer between default account and subaccounts.  Note the `_result` had to be reworked because the POST to `/profiles/transfer` doesn't have a JSON response.

### CODE REVIEW

File: pyexchange/coinbase.py [hunk 1]
```diff
@@ -318,13 +357,15 @@ def _http_unauthenticated(self, method: str, resource: str, body: dict):
...
-
-        try:
-            data = result.json()
-        except Exception:
-            raise Exception(f"Coinbase API invalid JSON response: {http_response_summary(result)}")
-
-        return data
+            raise RuntimeError(f"Coinbase API response: {http_response_summary(result)}")
+
+        if result.content and result.content != b'OK':
```
Comment (ith-harvey) [inline pyexchange/coinbase.py:365]: Hey Ed, could you explain why you're using the byte literal here? Is this the value stored in `result.content`? I assume this is some of the "reworking" you were referring to

File: tests/manual_test_coinbase.py [hunk 2]
```diff
@@ -15,14 +15,19 @@
...
 from pyexchange.coinbase import CoinbaseApi
 from pymaker import Address, Wad
 
+logging.basicConfig(format='%(asctime)-15s %(levelname)-8s %(message)s', level=logging.INFO)
+logging.getLogger('urllib3').setLevel(logging.INFO)
+logging.getLogger('urllib3.connectionpool').setLevel(logging.INFO)
+logging.getLogger("requests").setLevel(logging.INFO)
+logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.INFO)
 
+# ctor params: api_server: str, api_key: str, secret_key: str, password: str
```
Comment (ith-harvey) [inline tests/manual_test_coinbase.py:30]: not quite sure what `ctor` is referring to?

File: pyexchange/coinbase.py (hunk 1 above)
Comment (EdNoepel) [reply pyexchange/coinbase.py:365]: Yes, that's the success response when Coinbase doesn't return JSON.

File: tests/manual_test_coinbase.py (hunk 2 above)
Comment (EdNoepel) [reply tests/manual_test_coinbase.py:30]: Short for "constructor".  It's the parameters to pass.

**Pull request merged**